service/src/main/java/bio/terra/workspace/app/controller/ControlledAzureResourceApiController.java

```diff
@@ -257,39 +256,16 @@ public ResponseEntity<ApiCreateControlledAzureRelayNamespaceResult> createAzureR
         SamRethrow.onInterrupted(
             () -> getSamService().getUserEmailFromSam(userRequest), "getUserEmailFromSam");
 
-    logger.info(
```

Comment:
This has been refactored into the service-level SAS token generation method so we have centralized logging (i.e, I want to log when the blob copy flight mints a token as well and not have to dupe logging statements at that call site).

Reply:
@aherbst-broad it would be good to remove the userEmail variable ^